src/wifi.c

```diff
@@ -0,0 +1,84 @@
+#include <freertos/FreeRTOS.h>
+#include <freertos/event_groups.h>
+#include <esp_event_loop.h>
+#include <esp_wifi.h>
+#include <string.h>
+#include <stdlib.h>
+#include <nvs_flash.h>
+
+#include "wifi.h"
+
+#define CONNECTED_BIT BIT0
+
+static EventGroupHandle_t wifi_event_group;
+
+/**
+ * Handler for WiFi events
+ *
+ * @ctx: context pointer
+ * @event: Wifi event that occurred
+ *
+ * @return: esp_err_t 0 in every case
+ */
+static
+esp_err_t
+event_handler(void *ctx, system_event_t *event)
```

Comment:
```suggestion
event_handler(void * /*ctx*/, system_event_t *event)
```
if a parameter is not used, please remove the name binding to avoid unused warnings

Reply:
Your suggestion breaks the build.

including ctx this way does not yield a Warning

Reply:
yeah forgot it's c++ feature x.x